packages/gatsby/src/query/page-query-runner.js

```diff
@@ -221,3 +161,72 @@ const findDirtyIds = actions => {
   )
   return uniqDirties
 }
+
+const runInitialQueries = async activity => {
+  const pathnamesToRun = calcQueries(true)
+  if (pathnamesToRun.length === 0) {
+    return
+  }
+
+  const queryJobs = makeQueryJobs(pathnamesToRun)
+
+  const queue = queryQueue.makeBuild()
```

Comment:
Yeah, it makes sense within `query-queue.js` (since there's `makeBuild` and `makeDevelop`, but you're right that it's a bit confusing from outside. 

The other possibility I considered was to move `makeBuild` into `page-query-runner`, and call it `makeQueue`. That way we could pull `makeDevelop` directly into `src/commands/develop.js` (and just call it `makeQueryQueue`).

Reply:
I like what you're thinking. Moving `makeDevelop` in `src/commands/develop.js` probably makes the file more bloated which can also make it harder to understand.